plugins/modules/aci_epg_useg_attribute_simple_statement.py

```diff
@@ -0,0 +1,436 @@
+response:
+  description: The HTTP response from the APIC
+  returned: failure or debug
+  type: str
+  sample: OK (30 bytes)
+status:
+  description: The HTTP status from the APIC
+  returned: failure or debug
+  type: int
+  sample: 200
+url:
+  description: The HTTP url used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: https://10.11.12.13/api/mo/uni/tn-production.json
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec, aci_annotation_spec
+from ansible_collections.cisco.aci.plugins.module_utils.constants import USEG_ATTRIBUTE_MAPPING, OPERATOR_MAPPING
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(aci_annotation_spec())
+    argument_spec.update(
+        ap=dict(type="str", aliases=["app_profile", "app_profile_name"]),  # Not required for querying all objects
+        epg=dict(type="str", aliases=["epg_name"]),  # Not required for querying all objects
+        state=dict(type="str", default="present", choices=["absent", "present", "query"]),
+        tenant=dict(type="str", aliases=["tenant_name"]),  # Not required for querying all objects
+        parent_block_statements=dict(type="list", elements="str", aliases=["parent_blocks", "blocks"]),
+        name=dict(type="str", aliases=["useg_attribute_name"]),
+        type=dict(type="str", required=True, choices=list(USEG_ATTRIBUTE_MAPPING.keys()), aliases=["useg_attribute_type"]),
+        operator=dict(type="str", choices=list(OPERATOR_MAPPING.keys())),
+        category=dict(type="str", aliases=["custom_attribute"]),
+        value=dict(type="str"),
+        use_subnet=dict(type="bool"),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ["state", "absent", ["ap", "epg", "tenant", "name"]],
+            ["state", "present", ["ap", "epg", "tenant", "name"]],
+        ],
+    )
+
+    aci = ACIModule(module)
+
+    ap = module.params.get("ap")
+    epg = module.params.get("epg")
+    state = module.params.get("state")
+    tenant = module.params.get("tenant")
+    blocks = module.params.get("parent_block_statements")
+    name = module.params.get("name")
+    attribute_type = module.params.get("type")
+    value = module.params.get("value")
+    operator = module.params.get("operator")
+    category = module.params.get("category")
+    use_subnet = aci.boolean(module.params.get("use_subnet"))
+
+    # Excluding below classes from the module:
+    # fvProtoAttr:
+    #   Was used in AVS, but it is not longer in use.
+    # fvUsegBDCont:
+    #   Was part of a feature that allowed uSeg attributes to be applied at VRF (instead of BD) level.
+    #   It has been since deprecated and we no longer allow setting the scope at fvCtrn to scope-vrf.
+    #   This type of functionality has been replaced by the ESG feature.
```

Comment:
just a quick question on that: Shouldn't we add this in the documentation, under the note section?

Reply:
I added it on purpose in code only, since this should not be configured and didn't want to bloat the docs

Reply:
sounds good to me!